kafkaAdmin/resource_topic_create_test.go

```diff
@@ -97,6 +97,54 @@ resource "kafka_topic" "foobar" {
 }
 `
 
+func TestAccKafkaAdminTopicUpdate(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreventPostDestroyRefresh: true,
+		PreCheck:                  func() { testAccPreCheck(t) },
+		Providers:                 testAccProviders,
+		CheckDestroy:              testCheckTopicDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckKafkaTopicCreate,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTopicExists("kafka_topic.foo"),
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "name", "mytopic"),
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "partitions", "2"),
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "replication_factor", "3"),
+					resource.TestCheckResourceAttr(
+						"kafka_topic.foo", "retention_ms", "-1"),
```

Comment:
@comozo is this a valid value?

Reply:
@lenfree do you mean retention_ms should be retention.ms ?

Reply:
No, sorry for confusion. Please ignore this. I thought -1 is an invalid value but I was wrong.